app/renderer/src/components/Inspector/LocatedElements.jsx

```diff
@@ -46,7 +48,7 @@ const LocatedElements = (props) => {
       {locatedElements.length === 0 && (
         <Space className={InspectorStyles.spaceContainer} direction="vertical" size="small">
           <Row>
-            <i>{t('couldNotFindAnyElements')}</i>
+            <i>{t('couldNotFindAnyElements', {locatorTestStrategy, locatorTestValue})}</i>
           </Row>
```

Comment:
I think it's cleaner if the text string is not modified, and instead, additional lines are added with the locator details:
```suggestion
            <i>{t('couldNotFindAnyElements')}</i>
          </Row>
          <Row>
            {t('locatorStrategy')} {locatorTestStrategy}
            <br />
            {t('selector')} {locatorTestValue}
          </Row>
```
It would look like this:
![image](https://github.com/appium/appium-inspector/assets/37242620/4856f358-956c-4e39-8bce-ca5cfc2fdb66)

Reply:
@eglitise This look good to me as well. Let's also see what @mykola-mokhnach thinks?

Reply:
this might bite us in response for some locales where the order of words in acetone is different from English. Always prefer to use format specifier instead of hardcoded string catenation logic.

Reply:
@mykola-mokhnach I like the idea pointed out by @eglitise to show the locators details in new line to give better readability. Shall we proceed with the suggested design?

Reply:
I would still propose the above approach, because the order of words is the same one that is already used in the search panel modal landing page. Combining the text + strategy + selector in the same sentence makes it a lot more difficult to read, especially when italicized.

Reply:
@eglitise @mykola-mokhnach Initially i was not sure how the localisation has been implemented. Since we are using the translations as Label, it should work pretty much the same way as it is in the Search panel model.